docs/source/drush_commands.rst

```diff
@@ -11,6 +11,20 @@ dkan:dataset-info
 
 ~~~~~~
 
+dkan:datastore:reverse-dataset-lookup
+-----------------
+   Takes an argument of the table name and returns the dataset uuid.
+
+  **Arguments**
+
+  - **data_table_name** The datastore table name in the database.
```

Comment:
"data" is not useful in this context, can we just say table_name throughout this update?